### ISSUE TITLE
Software Product Vision Statement

### ISSUE THREAD

**Issue by angelajholden** [angelajholden/fiberandkraft#2]
For fiber enthusiasts who want to buy custom blend wool and hand dyed yarn, Fiber & Kraft curates and sells fiber for spinning, knitting and crocheting. Purchases are made from an ecommerce website that meets WCAG 2.2 accessibility standards and is GDPR compliant.

Our site has a minimal and aesthetically pleasing user interface for all devices and audiences. Unlike sites that have a terrible mobile experience and are not accessible to screen readers, we recognize and respect the needs of each member of the fiber community.

Our products are made from high quality wool that’s ethically sourced from midwest shepherds and milled locally in Minnesota. Unlike mills producing large volumes of woll from around the world, each skein of yarn or bump of fiber can be matched to a flock, ram, ewe or lamb.

**Comment by angelajholden** [comment]
Product Vision Statement is complete!
